zyte_spider_templates/spiders/ecommerce.py

```diff
@@ -23,6 +24,13 @@
 
 @document_enum
 class EcommerceCrawlStrategy(str, Enum):
+    default: str = "default"
```

Comment:
I think we have 2 options for the names.

1. Name it "default". It means that if we would change the default later, we'll be changing behavior of this strategy. So, all the spiders will get the new default. 
2. Name it something like "combined", "full+navigation", I don't know. This way, it can still be default, and the name is not going to be visible anywhere, but if we decide to change the default behavior, then the switch won't be automatic anymore.

I'm not sure what's better :) Leaning towards the first approach, but the second may also be worth considering. @BurnzZ @Gallaecio any thought?

By the way, if we go with "default" approach, I also wonder if we should name it something like "automatic" instead. It feels a bit off to have a strategy named "default" when there is an option to not have it as a default (e.g. in a template subclass); i think it's better to use a name which describes what the strategy does.

Reply:
I like the new name "automatic" as well. 👍 

I'm not quite sure about naming it like "full+navigation" since this can change in the future. For example, heuristics might be smarter to avoid crawling the subCategory links if it detects they are mostly FPs. Thus, the strategy now also includes "pagination_only"

Reply:
Renamed to "automatic" https://github.com/zytedata/zyte-spider-templates/pull/52/commits/dc947d1140bf4058ca1bf67dd60572364b0a7b64